adal/src/main/java/com/microsoft/aad/adal/BasicWebViewClient.java

```diff
@@ -480,13 +479,13 @@ public void run() {
         } else if (url.toLowerCase(Locale.US).startsWith(PLAY_STORE_INSTALL_PREFIX)) {
 
             view.stopLoading();
-            if (!(url.startsWith(PLAY_STORE_INSTALL_PREFIX + COMPANY_PORTAL_PROD_APP_PACKAGE_NAME))
+            if (!(url.startsWith(PLAY_STORE_INSTALL_PREFIX + COMPANY_PORTAL_APP_PACKAGE_NAME))
```

Comment:
Same as above : since we're launching playstore - these should be prod package name i think.

Reply:
With my changes there is now only one Company Portal and this is the actual prod CP....I've basically renamed the constants... the other one is actually called BROKER_HOST_PACKAGE_NAME